tests/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Build/SolutionBuildManagerTests.cs

```diff
@@ -0,0 +1,263 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.VisualStudio.ProjectSystem.VS.Build;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Threading;
+#pragma warning disable VSSDK005
```

Comment:
I can never remember these codes. It's nice to add the diagnostic's title as a comment:

```suggestion
#pragma warning disable VSSDK005 // Use the JoinableTaskContext singleton
```